users/views.py

```diff
@@ -1,3 +1,71 @@
-from django.shortcuts import render
+import json
+import re
+import bcrypt
+import jwt
 
-# Create your views here.
+from django.http      import JsonResponse, HttpResponse
+from django.views     import View
+from users.models     import User
+from my_settings      import SECRET_KEY
+from django.db.models import Q
+
+from users.utils      import LoginConfirm, is_valid, JWT_ALGORITHM, \
+                             PW_REGEX, EMAIL_REGEX,  NICKNAME_REGEX
+
+class SignupView(View): 
+    def post(self,request):
+        data = json.loads(request.body)
+        
+        try:
+            assert is_valid(data['email'],EMAIL_REGEX),       "INVALID_EMAIL_FORMAT"
+            assert is_valid(data['password'],PW_REGEX),       "INVALID_PW_FORMAT"
+            assert is_valid(data['nickname'],NICKNAME_REGEX), "INVALID_NICKNAME_FORMAT"
```

Comment:
```suggestion
            assert is_valid(data['email'], EMAIL_REGEX), "INVALID_EMAIL_FORMAT"
            assert is_valid(data['password'], PW_REGEX), "INVALID_PW_FORMAT"
            assert is_valid(data['nickname'], NICKNAME_REGEX), "INVALID_NICKNAME_FORMAT"
```
`,`콤마 뒤에는 간격 맞추시지 않으셔도 됩니다 :)

Reply:
넵 ! 수정했습니다 :+1: